packages/database/e2e/query/onChildMoved.modular.e2e.js

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright (c) 2016-present Invertase Limited & Contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this library except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+const { PATH, seed, wipe } = require('../helpers');
+
+const TEST_PATH = `${PATH}/on`;
+
+describe('onChildMoved', function () {
+  before(async function () {
+    await seed(TEST_PATH);
+  });
+  after(async function () {
+    await wipe(TEST_PATH);
+  });
+
+  // FIXME super flaky on ios simulator
+  it('should stop listening if ListeningOptions.onlyOnce is true', async function () {
+    if (device.getPlatform() === 'ios') {
+      this.skip();
```

Comment:
Missing v8 + modular ?

Reply:
V9 restructured it's API. Now instead of `on('child_moved')`, `on('child_added')`, etc. it's `onChildMoved()`, `onChildAdded()`, etc. Similarly, instead of `once('child_movied')`, it's `onChildMoved(ref, callback, { onlyOnce: true })`.